requirements/daemons/grs.txt

```diff
@@ -0,0 +1,3 @@
+https://github.com/Groestlcoin/electrum-grs/archive/refs/tags/v4.3.1.zip#egg=electrum-grs[crypto]
```

Comment:
Please change to the right hash like we discussed

Reply:
Done